charts/akeyless-api-gateway/templates/_helpers.tpl

```diff
@@ -108,6 +108,14 @@ Generate chart secret name
         {{ $.Release.Name }}-cache-secret
 {{- end -}}
 
+{{- define "akeyless-api-gw.clusterCacheEncryptionKeyExist" -}}
+        {{- if .Values.cachingConf.clusterCacheEncryptionKey -}}
+            {{ include "akeyless-api-gw.secretName" . }}
```

Comment:
{{ include "akeyless-api-gw.secretName" . }} ?? this is not related..
please check